### PR TITLE
First pull request, including all the basic functionality

### PR DESCRIPTION
**Pull request by ZhengyangChang**
Issue #5 
#### A simple stream status tool for pravega that can:
* Print out stream metadata and stream data in tier-1 and tier-2 storage.
* Print out metadata for cluster, transactions, stream segments.
* Print out any inconsistency found in different part of storage.

### CODE REVIEW

File: src/main/java/logs/DataFrame.java [hunk 1]
```diff
@@ -0,0 +1,92 @@
+package logs;
```
Comment (arvindkandhare) [inline src/main/java/logs/DataFrame.java]: Please add file, class and method errors as per Pravega code.

Comment (arvindkandhare) [inline src/main/java/logs/DataFrame.java]: Add checkstyle rules.

Comment (arvindkandhare) [inline src/main/java/logs/DataFrame.java]: Configure travis.

Comment (arvindkandhare) [inline src/main/java/logs/DataFrame.java]: Please create an issue and tie this PR to the issue.

**Comment by ZhengyangChang** [comment]
#5

**Pull request merged**